scripts/lind_compile

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
can we name this to "lind-clang" or alias? both?

Reply:
I chose a generic name, because the script runs `clang`, `wasm-opt` and `wasmtime compile`. Wouldn't `lind-clang` be misleading?

Would you mind adding aliases in follow-up PRs?

Reply:
Yeah thats fine with me.